src/api/lib/tasks/dev.rake

```diff
@@ -266,3 +293,20 @@ def copy_example_file(example_file)
     FileUtils.copy_file("#{example_file}.example", example_file)
   end
 end
+
+def request_for_staging(staging_project, maintainer_project, suffix)
+  requester = create(:confirmed_user, login: "requester_#{suffix}")
+  source_project = create(:project, name: "source_project_#{suffix}")
+  target_package = create(:package, name: "target_package_#{suffix}", project: maintainer_project)
+  source_package = create(:package, name: "source_package_#{suffix}", project: source_project)
+  request = create(
+    :bs_request_with_submit_action,
+    state: :new,
+    creator: requester,
+    target_package: target_package,
+    source_package: source_package,
+    staging_project: staging_project
+  )
+
+  request.reviews.each { |review| review.change_state(:accepted, 'Accepted') }
```

Comment:
What's the point of this?

Reply:
We needed a staging project that is in 'accepted' state. And for that all requests need to be accepted.

On second thought... we probably could / should handle this via a factory. :thinking:

Reply:
You guys confuse requests and reviews at a level that scares me.

If requests are accepted, they should not be in stagings any longer. And if requests don't have reviews because none was added, the staging project is acceptable.

Reply:
Where do you see an accepted request? ....

Reply:
Again: where are the reviews coming from you are accepting?

Reply:
Good question! There seems to be some code that creates them. Probably in the factories. Though I didn't find anything suspicious on first glance.

Reply:
Is it by chance by_group manager_group? If so, blindly accepting them without actually selecting it in the staging project might be fine for tests, but is surely not cool for development environment.

Reply:
@coolo in that cause I think we just wanted to bring the overall_state to ```:acceptable``` to be able to "merge" staging projects and be able to work on our card.. that's not a generic staging dashboard setup rather a specific one.

@bgeuken maybe we should rename the rake task to reflect what it really is?

Reply:
Yeah - and if you 'merge' staging projects you want a real staging project and not a fake one.

Reply:
Seems my message wasn't sent before:/

We started this rake task to have some test data that we can quickly set up on any machine. So that we have an easier time when switching the driver during our mob-programming session.
We don't claim that the data we create there are complete yet, and since we are using test factories to set this up, there might be some flaws we run into (which we are going to address).

But like @vpereira said the main goal here is to have a staging project in `:acceptable` state. This is needed so we can start implementing the `merge` feature.

Reply:
And as I said: if you start implementing merge on this staging projects, you will run into very strange problems you won't have if you set up these staging projects properly